test/cpp/end2end/xds/xds_end2end_test.cc

```diff
@@ -1393,9 +1415,10 @@ TEST_P(XdsServerSecurityTest, TestMtlsToFallback) {
   ASSERT_TRUE(backends_[0]->notifier()->WaitOnServingStatusChange(
       grpc_core::LocalIpAndPort(backends_[0]->port()), grpc::StatusCode::OK));
   SendRpc([this]() { return CreateMtlsChannel(); },
-          server_authenticated_identity_, client_authenticated_identity_);
+          RpcOptions().set_wait_for_ready(true), server_authenticated_identity_,
+          client_authenticated_identity_);
   SetLdsUpdate("", "", "", "", false);
-  SendRpc([this]() { return CreateInsecureChannel(); }, {}, {});
+  SendRpc([this]() { return CreateInsecureChannel(); }, RpcOptions(), {}, {});
```

Comment:
Shouldn't this use wait_for_ready?  It's a new channel, and it looks like you're using wait_for_ready in other cases in this situation, so I'm wondering why this one is different.

Reply:
Thanks!